paf-mvp-frontend/src/lib/paf-lib.ts

```diff
@@ -75,6 +80,32 @@ export class OneKeyLib implements IOneKeyLib {
     }
   }
 
+  parseCookieTTL(duration: string): number {
+    if (duration === undefined) {
+      this.log.Info('No cookie ttl was specified! Using default value:', DEFAULT_TTL_IN_SECONDS, 'seconds');
```

Comment:
[NIT] I think template strings are more "standard" (can be used in logs or elsewhere).
Ex:
```typesript
`No cookie ttl was specified! Using default value: ${DEFAULT_TTL_IN_SECONDS} seconds`
```
WDYT?